content/en/docs/getting-started/_index.html

```diff
@@ -0,0 +1,266 @@
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" data-toggle="tab" href="#linux">Linux</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" data-toggle="tab" href="#windows">Windows</a>
+        </li>
+      </ul>
+      <div class="tab-content" id="install-jx-tabs">
+        <div class="tab-pane fade show active" id="osx">
+        <h3>Using Homebrew</h3>
+          
+{{<highlight sh>}}
+brew install jenkins-x/jx/jx
+{{</highlight>}}
+          
+        <h3>Using Curl</h3>
+        <p>
+        Download the `.tar` file, unarchive it.
+        </p>
+
+{{<highlight sh>}}
+curl -L "https://github.com/jenkins-x/jx/releases/download/$(curl --silent https://api.github.com/repos/jenkins-x/jx/releases/latest | jq -r '.tag_name')/jx-darwin-amd64.tar.gz" | tar xzv "jx"
+{{</highlight>}}
+        
+        
+        Add  `jx` to your path
+        
+{{<highlight sh>}}
+sudo mv jx /usr/local/bin
+{{</highlight>}}
+        
+        Check if all is good!
+        
+{{<highlight sh>}}
+jx --version
+{{</highlight>}}
+        </div>
+        <div class="tab-pane fade" id="linux">
+        
+        &nbsp;  
+        Download the `.tar` file, and unarchive it in a directory where you can run the `jx` command.
+      
+{{<highlight sh>}}
+curl -L "https://github.com/jenkins-x/jx/releases/download/$(curl --silent https://api.github.com/repos/jenkins-x/jx/releases/latest | jq -r '.tag_name')/jx-linux-amd64.tar.gz" | tar xzv "jx"
+{{</highlight>}}
+      
+        or, if you don't have `jq` installed:
+      
+{{<highlight sh>}}
+curl -L "https://github.com/jenkins-x/jx/releases/download/$(curl --silent "https://github.com/jenkins-x/jx/releases/latest" | sed 's#.*tag/\(.*\)\".*#\1#')/jx-linux-amd64.tar.gz" | tar xzv "jx"
+{{</highlight>}}
+      
+        Move binary to your path
+      
+{{<highlight sh>}}
+sudo mv jx /usr/local/bin
+{{</highlight>}}
+      
+      Test all is good
+      
+{{<highlight sh>}}
+jx --version
+{{</highlight>}}
+      
+        </div>
+        <div class="tab-pane fade" id="windows">
+        &nbsp;
+      
+        <h3> Install Chocolatey using an Admin</h3>
+      
+      
+      1. Right-click menu:Start\[Command Prompt (Admin)\].
+      <br><br>
+      2. At the shell prompt, execute a `powershell.exe` command to download
+          and install the `choco` binary and set the installation path so that
+          the binary can be executed:
+
+{{<highlight sh>}}
+    @"%SystemRoot%\System32\WindowsPowerShell\v1.0\powershell.exe" -NoProfile -InputFormat None -ExecutionPolicy Bypass -Command "iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))" && SET "PATH=%PATH%;%ALLUSERSPROFILE%\chocolatey\bin"
+{{</highlight>}}
+
+      3.  Install Jenkins X using Chocolatey:
+      
+{{<highlight sh>}}
+  choco install jenkins-x
+{{</highlight>}}
+      
+      Upgrading Jenkins X via Chocolatey
```

Comment:
Why do we need the "upgrade" commands here? This is about getting started, right? If we need more information around Chocolatey we probably should have it somewhere else.